<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>849</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OS Translator II</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item row="4" column="2" colspan="2">
      <widget class="QSpinBox" name="simultaneousJobsSpinBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>128</number>
       </property>
       <property name="value">
        <number>2</number>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Processor Cores</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QPushButton" name="deleteTaskPushButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="newTaskPushButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>New</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Import Tasks</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="4">
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="4">
      <widget class="QListWidget" name="tasksListWidget"/>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="1" colspan="2">
      <widget class="QLineEdit" name="labelLineEdit">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Mode</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Dataset</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="2">
      <widget class="QComboBox" name="postgisConnectionComboBox"/>
     </item>
     <item row="14" column="1">
      <widget class="QLineEdit" name="inputLineEdit">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Label</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1" colspan="2">
      <widget class="QComboBox" name="modeComboBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>Create or Replace</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Append</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QComboBox" name="datasetComboBox"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Connection</string>
       </property>
      </widget>
     </item>
     <item row="14" column="0">
      <widget class="QLabel" name="inputLabel">
       <property name="text">
        <string>Input</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Schema</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0" colspan="3">
      <widget class="QCheckBox" name="batchModeCheckBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Recursively process a directory</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="1" colspan="2">
      <widget class="QLineEdit" name="destSchema"/>
     </item>
     <item row="14" column="2">
      <widget class="QPushButton" name="browsePushButton">
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="15" column="1">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="0" colspan="3">
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="10" column="0" colspan="3">
      <widget class="Line" name="line_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="3">
      <widget class="QCheckBox" name="createIndicesCheckBox">
       <property name="text">
        <string>Create indices</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="7" column="0" colspan="3">
      <widget class="QCheckBox" name="removeDuplicatesCheckBox">
       <property name="toolTip">
        <string/>
       </property>
       <property name="statusTip">
        <string/>
       </property>
       <property name="text">
        <string>Remove duplicates (chunk artefacts)</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="8" column="0" colspan="3">
      <widget class="QCheckBox" name="addOsStylingFieldsCheckBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Add OS styling fields</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0" colspan="3">
      <widget class="QCheckBox" name="applyDefaultOsStyleCheckBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Apply default OS style</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="3" colspan="3">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Fields</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QTreeWidget" name="fieldsTreeWidget">
       <property name="minimumSize">
        <size>
         <width>354</width>
         <height>0</height>
        </size>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::NoSelection</enum>
       </property>
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="helpPushButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Help</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="aboutPushButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>About</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="2" colspan="2">
    <widget class="QProgressBar" name="progressBar">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="maximum">
      <number>99</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QLabel" name="statusLabel">
     <property name="text">
      <string>Status label</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tasksListWidget</tabstop>
  <tabstop>deleteTaskPushButton</tabstop>
  <tabstop>newTaskPushButton</tabstop>
  <tabstop>simultaneousJobsSpinBox</tabstop>
  <tabstop>labelLineEdit</tabstop>
  <tabstop>datasetComboBox</tabstop>
  <tabstop>postgisConnectionComboBox</tabstop>
  <tabstop>modeComboBox</tabstop>
  <tabstop>destSchema</tabstop>
  <tabstop>batchModeCheckBox</tabstop>
  <tabstop>inputLineEdit</tabstop>
  <tabstop>browsePushButton</tabstop>
  <tabstop>buttonBox</tabstop>
  <tabstop>fieldsTreeWidget</tabstop>
  <tabstop>helpPushButton</tabstop>
  <tabstop>aboutPushButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>839</x>
     <y>343</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>839</x>
     <y>343</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>browsePushButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>browseForInput()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>476</x>
     <y>295</y>
    </hint>
    <hint type="destinationlabel">
     <x>478</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>simultaneousJobsSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Dialog</receiver>
   <slot>storeSettings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>243</x>
     <y>312</y>
    </hint>
    <hint type="destinationlabel">
     <x>353</x>
     <y>314</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>datasetComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>Dialog</receiver>
   <slot>storeSettings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>409</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>334</x>
     <y>302</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>postgisConnectionComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>Dialog</receiver>
   <slot>storeSettings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>476</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>323</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>modeComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>Dialog</receiver>
   <slot>storeSettings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>476</x>
     <y>116</y>
    </hint>
    <hint type="destinationlabel">
     <x>401</x>
     <y>332</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>destSchema</sender>
   <signal>textChanged(QString)</signal>
   <receiver>Dialog</receiver>
   <slot>storeSettings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>476</x>
     <y>142</y>
    </hint>
    <hint type="destinationlabel">
     <x>375</x>
     <y>288</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>datasetComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>Dialog</receiver>
   <slot>updateFieldsList()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>360</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>235</x>
     <y>118</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>helpPushButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>helpPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>343</y>
    </hint>
    <hint type="destinationlabel">
     <x>303</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>aboutPushButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>aboutPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>165</x>
     <y>343</y>
    </hint>
    <hint type="destinationlabel">
     <x>83</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>createIndicesCheckBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>Dialog</receiver>
   <slot>storeSettings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>306</x>
     <y>153</y>
    </hint>
    <hint type="destinationlabel">
     <x>316</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>removeDuplicatesCheckBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>Dialog</receiver>
   <slot>storeSettings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>364</x>
     <y>180</y>
    </hint>
    <hint type="destinationlabel">
     <x>355</x>
     <y>171</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>datasetComboBox</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>Dialog</receiver>
   <slot>updateImportOptions()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>343</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>310</x>
     <y>32</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>applyDefaultOsStyleCheckBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>Dialog</receiver>
   <slot>applyDefaultOsStyleCheckBoxChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>444</x>
     <y>227</y>
    </hint>
    <hint type="destinationlabel">
     <x>425</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addOsStylingFieldsCheckBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>Dialog</receiver>
   <slot>storeSettings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>293</x>
     <y>203</y>
    </hint>
    <hint type="destinationlabel">
     <x>256</x>
     <y>207</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>browseForInput()</slot>
  <slot>storeSettings()</slot>
  <slot>updateFieldsList()</slot>
  <slot>helpPressed()</slot>
  <slot>aboutPressed()</slot>
  <slot>updateImportOptions()</slot>
  <slot>applyDefaultOsStyleCheckBoxChanged(int)</slot>
 </slots>
</ui>
